proc/test/support.go

```diff
@@ -48,13 +50,23 @@ func BuildFixture(name string) Fixture {
 	path := filepath.Join(fixturesDir, name+".go")
 	tmpfile := filepath.Join(os.TempDir(), fmt.Sprintf("%s.%s", name, hex.EncodeToString(r)))
 
+	buildFlags := []string{"build"}
+	if runtime.GOOS == "windows" {
+		// Work-around for https://github.com/golang/go/issues/13154
+		buildFlags = append(buildFlags, "-ldflags=-linkmode internal")
+	}
+	buildFlags = append(buildFlags, "-gcflags=-N -l", "-o", tmpfile, path)
+
 	// Build the test binary
-	if err := exec.Command("go", "build", "-gcflags=-N -l", "-o", tmpfile, path).Run(); err != nil {
+	if err := exec.Command("go", buildFlags...).Run(); err != nil {
 		fmt.Printf("Error compiling %s: %s\n", path, err)
 		os.Exit(1)
 	}
 
 	source, _ := filepath.Abs(path)
+	if runtime.GOOS == "windows" {
```

Comment:
Same answer as above - happy to change to use the code without a platform test if that's preferred.